fastlane_core/lib/fastlane_core/configuration/configuration_file.rb

```diff
@@ -43,7 +43,7 @@ def initialize(config, path, block_for_missing, skip_printing_values = false)
 
       begin
         # rubocop:disable Security/Eval
-        puts(eval(content)) # this is okay in this case
```

Comment:
Huh, any idea why it was there in the first place?

Reply:
Something I missed when reviewing #11946 🙃

Reply:
@taquitos It was added by https://github.com/matsune/fastlane/commit/7d12cdabc8850225daa44a0a66477e94d1b0dd78

Reply:
I'm guessing it was added for debugging purposes and was left there by mistake.

Reply:
Oh, that makes sense. I suspected that to be the case. Thanks for checking!